packages/protocol/test/governance/lockedgold.ts

```diff
@@ -412,4 +413,37 @@ contract('LockedGold', (accounts: string[]) => {
       })
     })
   })
+
+  describe('#slash', () => {
+    const value = 1000
+    const reporter = accounts[2]
+
+    beforeEach(async () => {
+      // @ts-ignore: TODO(mcortesi) fix typings for TransactionDetails
+      await lockedGold.lock({ value })
+      await lockedGold.addSlasher(account)
+    })
+
+    describe('when the account is slashed for all of its locked gold', () => {
+      beforeEach(async () => {
+        await lockedGold.slash(
+          account,
+          value,
+          reporter,
+          value / 2,
+          [NULL_ADDRESS],
+          [NULL_ADDRESS],
+          [0]
+        )
+      })
+
+      it("should reduce account's locked gold balance to 0", async () => {
+        assertEqualBN(await lockedGold.getAccountNonvotingLockedGold(account), 0)
+      })
+
+      it("should increase the reporter's locked gold", async () => {
```

Comment:
Add test that the community fund receives gold

Reply:
How can I test the governance contract's balance? Traditional methods like `web3.eth.getBalance` gives me `0`, but using our contracts gives me issues with addresses not being registered.

Reply:
The code is `getGoldToken().transfer(communityFund, penalty.sub(reward));`
so I think you can test mock locked gold (just have to change it so that transfer actually does something)